frontend/src/metabase/admin/embedding/components/ThemeListing/EmbeddingThemeCard.tsx

```diff
@@ -0,0 +1,69 @@
+import { t } from "ttag";
+
+import { ActionIcon, Card, Group, Icon, Menu, Text } from "metabase/ui";
+import type { EmbeddingTheme } from "metabase-types/api/embedding-theme";
+
+import { EmbeddingThemeCardPreview } from "./EmbeddingThemeCardPreview";
+
+export function EmbeddingThemeCard({
+  theme,
+  onEdit,
+  onDuplicate,
+  onDelete,
+}: {
+  theme: Omit<EmbeddingTheme, "settings">;
+  onEdit: (id: number) => void;
+  onDuplicate: (id: number) => void;
+  onDelete: (id: number) => void;
+}) {
+  return (
+    <Card
+      p={0}
+      withBorder
+      onClick={() => onEdit(theme.id)}
+      style={{ cursor: "pointer" }}
+    >
+      <EmbeddingThemeCardPreview />
+
+      <Group align="center" justify="space-between" px="md" py="sm">
+        <Text fz="lg">{theme.name}</Text>
+
+        <EmbeddingThemeActionMenu
+          theme={theme}
+          onDuplicate={onDuplicate}
+          onDelete={onDelete}
+        />
+      </Group>
+    </Card>
+  );
+}
+
+const EmbeddingThemeActionMenu = ({
+  theme,
+  onDuplicate,
+  onDelete,
+}: {
+  theme: Omit<EmbeddingTheme, "settings">;
+  onDuplicate: (id: number) => void;
+  onDelete: (id: number) => void;
+}) => {
+  return (
+    <Menu position="bottom-end">
+      <Menu.Target>
+        <ActionIcon onClick={(e) => e.stopPropagation()}>
```

Comment:
This is so we don't trigger `onEdit` if we click on the action icon. This pattern is also used in collection cards for example:

<img width="912" height="682" alt="Image" src="https://github.com/user-attachments/assets/03b356e5-a5e2-40ba-bca5-fdc16feeb454" />